provider/terraform/utils/iam_billing_account.go

```diff
@@ -0,0 +1,105 @@
+package google
+
+import (
+	"fmt"
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform/helper/schema"
+	"google.golang.org/api/cloudbilling/v1"
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+var IamBillingAccountSchema = map[string]*schema.Schema{
+	"resource": {
```

Comment:
What do you think about renaming this "billing_account_id", for clarity?

Reply:
Done!